### PR TITLE
Create truth_table_and_or_not.py

### PR DESCRIPTION
**Pull request by its-100rabh**
### Describe your change:
Implemented a Python program to display the truth table of AND , OR and NOT.

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [ ] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [ ] All new algorithms include at least one URL that points to Wikipedia or another similar explanation.
* [ ] If this pull request resolves one or more open issues then the description above includes the issue number(s) with a [closing keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue): "Fixes #ISSUE-NUMBER".

### CODE REVIEW

File: boolean_algebra/truth_table_and_or_not.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
+def truth_table_and() -> None:
```
[the remark this answers is not in the document]
Comment (its-100rabh) [reply boolean_algebra/truth_table_and_or_not.py:1]: It's a display of the truth table, hence doctests are not required.

File: boolean_algebra/truth_table_and_or_not.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    print("Truth table of AND:\n")
+    print("a\tb\ta AND b")
+    print("-----------------")
+    for a in [0, 1]:
+        for b in [0, 1]:
+            c = a and b
+            print(f"{a}\t{b}\t{c}")
+
+
+def truth_table_or() -> None:
```
[the remark this answers is not in the document]
Comment (its-100rabh) [reply boolean_algebra/truth_table_and_or_not.py:24]: It's a display of the truth table, hence doctests are not required.

File: boolean_algebra/truth_table_and_or_not.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    print("\nTruth table of OR:\n")
+    print("a\tb\ta OR b")
+    print("----------------")
+    for a in [0, 1]:
+        for b in [0, 1]:
+            c = a or b
+            print(f"{a}\t{b}\t{c}")
+
+
+def truth_table_not() -> None:
```
[the remark this answers is not in the document]
Comment (its-100rabh) [reply boolean_algebra/truth_table_and_or_not.py:47]: It's a display of the truth table, hence doctests are not required.

**Comment by tianyizheng02** [comment]
Please read our contributing guidelines: algorithmic functions shouldn't print or have other side effects. We also already have AND, OR, and NOT gate implemented in the same directory.

**Pull request closed without merging**